vnext/src-win/Libraries/Text/TextProps.windows.js

```diff
@@ -219,7 +220,7 @@ export type TextProps = $ReadOnly<{
    *
    * See https://reactnative.dev/docs/text#selectioncolor
    */
-  selectionColor?: ?string,
+  selectionColor?: ?(number | NativeColorValue),
```

Comment:
Did we have to add NativeColorValue here or did it come from upstream?

Reply:
I had to add NativeColorValue due to this error: https://dev.azure.com/ms/react-native-windows/_build/results?buildId=593699&view=logs&j=61ec2c8a-143b-53b9-4800-dbbf77625938&t=052bb252-87e1-5f88-56de-95e16920c7f2&l=943. 

The upstream TextProps.js file has `selectionColor?: ?string`, so should we keep our implementation the same and ignore the linting issue, or should I keep the fork, and if so, log an issue?

Reply:
I think some other change you did might have fixed this because I am not seeing it locally. Try removing it and see if it still fails.

Reply:
If I remove the change, I start getting the linting error locally again:
```
Error ---------------------------------------------------------------------------- Libraries/Text/Text.windows.js:401:29

Cannot create `NativeText` element because in property `selectionColor`: [incompatible-type]
 - Either string [1] is incompatible with number [2].
 - Or string [1] is incompatible with `NativeColorValue` [3].

   Libraries/Text/Text.windows.js:401:29
   401|             selectionColor={selectionColor}
                                    ^^^^^^^^^^^^^^

References:
   Libraries/Text/TextProps.windows.js:223:21
   223|   selectionColor?: ?string,
                            ^^^^^^ [1]
   Libraries/StyleSheet/processColor.js:18:35
    18| export type ProcessedColorValue = number | NativeColorValue;
                                          ^^^^^^ [2]
   Libraries/StyleSheet/processColor.js:18:44
    18| export type ProcessedColorValue = number | NativeColorValue;
                                                   ^^^^^^^^^^^^^^^^ [3]
```

Reply:
I think you just need to rerun yarn! That line in your code doesn't exist anymore selectionColor={selectionColor} was changed to selectionColor={_selectionColor} :)